tests/cupy_tests/linalg_tests/test_einsum.py

```diff
@@ -269,9 +269,6 @@ def test_too_many_dims3(self, xp):
     {'shape_a': (2, 3), 'subscripts': 'ij->...ij'},  # do nothing
     {'shape_a': (2, 3), 'subscripts': 'ij...->ij'},  # do nothing
     {'shape_a': (2, 3), 'subscripts': 'i...j->ij'},  # do nothing
-
-    {'shape_a': (), 'subscripts': ''},  # do nothing
-    {'shape_a': (), 'subscripts': '->'},  # do nothing
```

Comment:
I think these patterns should be supported.

```
>>> numpy.__version__
'1.17.0'
>>> numpy.einsum('', numpy.ones(()))
1.0
>>> numpy.einsum('->', numpy.ones(()))
1.0
```

Reply:
The test failure is due to `numpy_cupy_allclose` `calling numpy.nonzero` and these patterns should be supported as you say. Thanks.